Commandant/ArgumentParser.swift

```diff
@@ -86,6 +86,15 @@ public final class ArgumentParser {
 		}
 	}
 
+	private init(rawArguments: [RawArgument]) {
```

Comment:
It seems weird to manually implement copy semantics. How about making this a `struct ` instead?